vividus/src/main/java/org/vividus/runner/GenericRunner.java

```diff
@@ -51,11 +51,7 @@
     private final Embedder embedder;
     private final IBatchedPathFinder batchedPathFinder;
 
-    protected static void setEmbedderBeanName(String embedderBeanName)
-    {
-        GenericRunner.embedderBeanName = embedderBeanName;
-    }
-
+    public GenericRunner()
```

Comment:
Please try to run as JUnit in Eclipse at first

Reply:
![image](https://user-images.githubusercontent.com/17816729/106482664-b9e09880-64be-11eb-9fe0-8af5ac239595.png)
Looks working